appveyor.yml

```diff
@@ -2,7 +2,7 @@ configuration: Debug
 image: Visual Studio 2017
 
 environment:
-  app_version: '7.5.0'
+  app_version: '8.0'
```

Comment:
@Yuri05 Version is now going to be Major.Minor.Build e.g.

Version 8
Update 1
Build xxx (comes from Appveyor)